testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -1084,7 +1082,9 @@ impl BitcoinRegtestController {
         // Serialize the payload
         let op_bytes = {
             let mut bytes = self.config.burnchain.magic_bytes.as_bytes().to_vec();
-            payload.consensus_serialize(&mut bytes).ok()?;
+            payload
+                .consensus_serialize(&mut bytes)
+                .map_err(|_| BurnchainControllerError::SerializerError)?;
```

Comment:
If there's a serialize error shouldn't we propagate some information on how exactly serialization failed?

Reply:
👍 added in d85c52e040ae704c41f948812d06d38d9a64be63.